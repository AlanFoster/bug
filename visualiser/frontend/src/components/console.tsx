import * as React from "react";

interface Props {
    readonly logs: string[];
}

export const Console: React.FC<Props> = function ({logs}) {
    return (
        <div>
            <div>Console ({logs.length} logs)</div>
            <pre>
                {logs.map(function (log, index) {
                    return <div key={index}>{log}</div>
                })}
            </pre>
        </div>
    )
};
