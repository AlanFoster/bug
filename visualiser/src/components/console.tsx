import * as React from "react";

export const Console = function () {
    return (
        <React.Fragment>
            Console
        </React.Fragment>
    )
};
