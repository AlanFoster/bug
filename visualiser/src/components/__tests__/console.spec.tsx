import * as React from "react";
import renderer from "react-test-renderer"
import { Console } from '../console';

describe("Console", function () {
    it("renders successfully", function () {
        const tree = renderer.create(<Console/>);

        expect(tree).toMatchSnapshot();
    });
});
