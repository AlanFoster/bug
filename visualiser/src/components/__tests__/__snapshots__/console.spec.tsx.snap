// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Console renders successfully 1`] = `"Console"`;
